lambdas/call-ticf-cri/src/test/java/uk/gov/di/ipv/core/callticfcri/pact/ticfCri/ContractTest.java

```diff
@@ -0,0 +1,631 @@
+import uk.gov.di.ipv.core.library.persistence.item.IpvSessionItem;
+import uk.gov.di.ipv.core.library.service.ConfigService;
+import uk.gov.di.ipv.core.library.verifiablecredential.validator.VerifiableCredentialJwtValidator;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.sql.Date;
+import java.text.ParseException;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.util.List;
+import java.util.Map;
+
+import static au.com.dius.pact.consumer.dsl.LambdaDsl.newJsonBody;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.when;
+import static uk.gov.di.ipv.core.library.domain.CriConstants.TICF_CRI;
+
+@Disabled("PACT tests should not be run in build pipelines at this time")
+@ExtendWith(PactConsumerTestExt.class)
+@ExtendWith(MockitoExtension.class)
+@PactTestFor(providerName = "TicfCriProvider")
+@MockServerConfig(hostInterface = "localhost", port = "1234")
+class ContractTest {
+    private static final String API_PATH = "/risk-assessment";
+    private static final String PRIVATE_API_KEY = "dummyApiKey";
+    private static final String PRIVATE_TICF_SIGNING_KEY =
+            """
+                    {"kty":"EC","d":"OXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthU","crv":"P-256","x":"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM","y":"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04"}
+            """;
+    private static final String APPLICATION_JSON = "application/json";
+    private static final Clock CURRENT_TIME =
+            Clock.fixed(Instant.parse("2099-01-01T00:00:00.00Z"), ZoneOffset.UTC);
+    private static JwtTestHelper dvlaVcJwtHelper;
+    private static JwtTestHelper passportVcJwtHelper;
+    private static JwtTestHelper dvlaWithCiVcJwtHelper;
+    private static JwtTestHelper emptyTicfVcJwtHelper;
+    private static JwtTestHelper interventionTicfVcJwtHelper;
+    private static JwtTestHelper noInterventionTicfVcJwtHelper;
+    private static JwtTestHelper noInterventionWithWarningsTicfVcJwtHelper;
+
+    @Mock ConfigService mockConfigService;
+
+    @BeforeAll
+    public static void setup() throws IOException {
+        dvlaVcJwtHelper = new JwtTestHelper("dvlaVc");
+        passportVcJwtHelper = new JwtTestHelper("passportVc");
+        dvlaWithCiVcJwtHelper = new JwtTestHelper("dvlaWithCiVc");
+        emptyTicfVcJwtHelper = new JwtTestHelper("ticfVc/empty");
+        interventionTicfVcJwtHelper = new JwtTestHelper("ticfVc/intervention");
+        noInterventionTicfVcJwtHelper = new JwtTestHelper("ticfVc/noIntervention");
+        noInterventionWithWarningsTicfVcJwtHelper =
+                new JwtTestHelper("ticfVc/noInterventionWithWarnings");
+    }
+
+    @Pact(provider = "TicfCriProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact requestForUserWithNoInterventions(PactDslWithProvider builder) {
```

Comment:
Naming these is tricky, but I think I'd go for something more like `validPassportVcReturnsVcWithNoInterventions()`